src/TeamSyncer.ts

```diff
@@ -392,28 +392,23 @@ export class TeamSyncer {
                     text: `Hint: To bridge to Matrix, run the \`/invite @${user!.name}\` command in this channel.`,
                     channel: channelItem.id,
                 });
-            } catch (ex) {
+            } catch {
```

Comment:
Wonder if we should have logged the exception at debug level.

Reply:
Added a `log.debug` of the error.